blog/r-tidyfinance-0-1-0/index.qmd

```diff
@@ -0,0 +1,185 @@
+---
+title: "tidyfinance 0.1.0"
+author:
+  - name: Christoph Scheuch
+    url: https://christophscheuch.github.io/
+    affiliations:
+      - name: wikifolio Financial Technologies AG
+  - name: Stefan Voigt
+    url: https://www.voigtstefan.me/
+    affiliations:
+      - name: University of Copenhagen
+      - name: Danish Finance Institute
+  - name: Patrick Weiss
+    url: https://sites.google.com/view/patrick-weiss
+    affiliations:
+      - name: Reykjavik University
+      - name: WU Vienna University of Economics and Business
+date: "2023-03-07"
+description: tidyfinance 0.1.0 is now on CRAN. Discover what this release includes. 
+image: thumbnail.png
+image-alt: An image depicting a busy warehouse scene with a single cardboard box on a conveyor belt, labeled 'tidyfinance' and no other text or stickers. In the background, workers in safety vests are sorting and scanning other packages amidst shelves filled with various boxes and logistics equipment, under a well-lit, organized, and efficient atmosphere. Created with DALL-E 3.
+categories: 
+  - Data
+  - R
+--- 
+
+We are happy to annouce the initial release of the `tidyfinance` R package [on CRAN](https://cran.r-project.org/web/packages/tidyfinance/index.html). The package contains a set of helper functions for empirical research in financial economics, addressing a variety of topics covered in [Tidy Finance with R](../../r/index.qmd) (TFWR). The package is designed to provide shortcuts for issues extensively discussed in the book, facilitating easier application of its concepts. If you want to inspect the details of the package or propose new features, feel free to visit the package repository on [Github](https://github.com/tidy-finance/r-tidyfinance).
+
+In this blog post, we demonstrate the features of the initial release. We decided to focus on functions that allow you to download the data that we use in TFWR. 
+
+## Install the package
+
+You can install the released version of `tidyfinance` from CRAN via:
+
+```{r}
+#| eval: false
+install.packages("tidyfinance")
+```
+
+You can install the development version of tidyfinance from GitHub using the `pak` package:
+
+```{r}
+#| eval: false
+pak::pak("tidy-finance/r-tidyfinance")
+```
+
+## Download data 
+
+Let's start by loading the package
+
+```{r}
+library(tidyfinance)
+```
+
+The main function is `download_data(type, start_date, end_date)` with supported type:
+```{r}
+list_supported_types()
+```
+
+So, for instance, if you want to download monthly Fama-French Three-Factor data, you can call:
+```{r}
+#| message: false
+download_data("factors_ff3_monthly", "2020-01-01", "2020-12-31")
+```
+Under the hood, the function uses the `frenchdata` package and applies some cleaning steps, as in TFWR. If you haven't installed `frenchdata` yet, you'll get prompted to install it first before you can download this specific data type. 
```

Comment:
link to documentation: https://cran.r-project.org/web/packages/frenchdata/index.html